test-utils/runtime/transaction-pool/src/lib.rs

```diff
@@ -52,9 +53,30 @@ impl std::error::Error for Error {
 	}
 }
 
+pub enum IsBestBlock {
+	Yes,
+	No,
+}
+
+impl IsBestBlock {
+	pub fn is_best(&self) -> bool {
+		matches!(self, Self::Yes)
+	}
+}
+
+impl From<bool> for IsBestBlock {
+	fn from(is_best: bool) -> Self {
+		if is_best {
+			Self::Yes
+		} else {
+			Self::No
+		}
+	}
+}
+
 #[derive(Default)]
 pub struct ChainState {
-	pub block_by_number: BTreeMap<BlockNumber, Vec<Block>>,
+	pub block_by_number: BTreeMap<BlockNumber, Vec<(Block, IsBestBlock)>>,
```

Comment:
I feel like instead of doing this we should simply store `best_block_number: BlockNumber` in that struct. And update it every time we get notification about new best block being imported or re-org happened.

Reply:
This is test code, we don't have any notifications about importing blocks there :D

Reply:
Fair enough :)